src/saga/mergeUtils.js

```diff
@@ -1,167 +1,14 @@
 import { conflictType } from "../constants";
 import { numToChar } from "./sagaUtils";
 
-function checkEmpty(row) {
-    const filteredRow = row.filter(element => element != "");
-    return filteredRow.length === 0; 
-}
-
-function handleOriginUndefinedRow(aRow, bRow, sheetName, possibleConflictType, rowIndex) {
-
-    // If only one version made edits to the row, then no conflict exists
-    if (aRow === undefined || checkEmpty(aRow)) {
-        return {result: bRow, conflicts: []};
-    }
-
-    if (bRow == undefined || checkEmpty(bRow)) {
-        return {result: aRow, conflicts: []};
-    }
-
-    let row = []
-    let conflicts = []
-    const maxLength = Math.max(aRow.length, bRow.length);
-    for (var i = 0; i < maxLength; i++) {
-        const aElement = aRow[i];
-        const bElement = bRow[i];
-
-        //TODO: If the entire rows are different, then throw a conflictType.ROW error. 
-
-        // If aRow does not contain a value at the cell, then take the b value which can't be undefined
-        if (aElement === undefined || aElement === "") {
-            // TODO: Make sure that bElement is not undefined
-            row.push(bElement)
-        } else if (bElement === undefined || bElement === "") {
-            // If bRow does not contain a value at the cell, then take the a value which can't be undefined
-
-            // TODO: Make sure that aElement is not undefined
-            row.push(aElement)
-        } else if (aElement === bElement) {
-            // If both cells are updated to the same value, then no conflict
-            row.push(aElement)
-        } else if (aElement !== bElement) {
-            // If the cells are different, then create a merge conflict and default to the value in a
-            const columnName = numToChar(i + 1);
-            const excelRow = rowIndex + 1
-            const cell = columnName + excelRow;
-                
-            row.push(aElement)
-            conflicts.push({
-                conflictType: conflictType.CELL,
-                sheet: sheetName,
-                cellOrRow: cell,
-                a: aElement,
-                b: bElement,
-                o: ""
-            })
-        }
-    }
-    return {result: row, conflicts: conflicts}
-}
-
-function handleOriginUndefinedElement(aElement, bElement, sheetName, possibleConflictType, cell) {
-
-    if (aElement === undefined) {
-        return {result: bElement, conflicts: []};
-    }
-
-    if (bElement === undefined) {
-        return {result: aElement, conflicts: []};
-    }
-
-    if (aElement === bElement) {
-        return {result: aElement, conflicts: []};
+// This makes it so we can "square everything up..."
+export class ValueWrapper {
```

Comment:
The functionality that this class provides is great -- not having to handle the undefined case makes life much simpler. Does this really deserve its own class though? It's just one function, we are just returning the value at an index or null. Seems strange to me that its a class, but lmk the reason for your decision. Seems like a distinction interesting to learn. 

Maybe its just me going back my CIS120 days when I refused to use classes.

Reply:
Great question. I think you're right that this shouldn't be a class. It's not much code, so I vote we keep it for now, but I think you're right this would be better as some function (or just in-lining the one line of code).